### PR TITLE
Document how to remove metadata from an Object

### LINKED ISSUE #4010: Undocumented way to remove metadata variants in Object
**Your Godot version:**
3.2

**Issue description:**
The `Object` class documentation does not state that using `null` in a `set_meta` call deletes the specified metadata entry.

```gdscript
#!/usr/bin/godot -s
extends SceneTree

func _init():
	print("Unmodified Meta List")
	print(get_meta_list())
	print("New test meta")
	set_meta('test', true)
	print("Modified Meta List -- added: test")
	print(get_meta_list())
	set_meta('test', null)
	print("Modified Meta List -- removed: test")
	print(get_meta_list())
	quit()
```

This should be documented.

**URL to the documentation page (if already existing):**
https://docs.godotengine.org/en/stable/classes/class_object.html

### PR DESCRIPTION
**Pull request by Calinou**
This closes https://github.com/godotengine/godot-docs/issues/4010.

### CODE REVIEW

**Comment by YeldhamDev** [comment]
Isn't this the job of `remove_meta()`?

**Review by NathanLovato — APPROVED** [review]

**Comment by akien-mga** [comment]
> Isn't this the job of `remove_meta()`?

Yeah, this should probably be documented as a side-effect of assigning `null`, and not as the way to remove metadata (for which `[method remove_meta]` can be linked).

File: doc/classes/Object.xml [hunk 1]
```diff
@@ -464,7 +464,7 @@
 			<argument index="1" name="value" type="Variant">
 			</argument>
 			<description>
-				Adds or changes a given entry in the object's metadata. Metadata are serialized, and can take any [Variant] value.
+				Adds, changes or removes a given entry in the object's metadata. Metadata are serialized and can take any [Variant] value. To remove a given entry from the object's metadata, use [method remove_meta].Metadata is also considered removed if its value is set to [code]null[/code]. This means you can also use [code]set_meta("name", null)[/code] to remove metadata for [code]"name"[/code].
```
Comment (akien-mga) [inline doc/classes/Object.xml:467]:
```suggestion
				Adds, changes or removes a given entry in the object's metadata. Metadata are serialized and can take any [Variant] value.
				To remove a given entry from the object's metadata, use [method remove_meta]. Metadata is also removed if its value is set to [code]null[/code]. This means you can also use [code]set_meta("name", null)[/code] to remove metadata for [code]"name"[/code].
```

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.3.

**Pull request merged**